yarn-project/ethereum/src/l1_tx_utils.ts

```diff
@@ -359,6 +366,9 @@ export class L1TxUtils {
       priorityFee = (priorityFee * (100n + (gasConfig.priorityFeeBumpPercentage || 0n))) / 100n;
     }
 
+    // Add priority fee to maxFeePerGas
+    maxFeePerGas += priorityFee;
```

Comment:
This should happen before the `maxFeePerGas = maxFeePerGas > minMaxFee ? maxFeePerGas : minMaxFee`, since the `minMaxFee` already has the priority fee component accounted for. 

Otherwise, if `maxFeePerGas` gets set to `minMaxFee` and then we add `priorityFee` on top of it, we'd be accounting for the `priorityFee` twice.

Reply:
right ofc 🤦